### DISCUSSION TITLE
WARN  Error parsing JSON at C:\Users\\scoop\buckets\extras\bucket\avsUSERpmod.json

### DISCUSSION THREAD

**Discussion by gerroon** [ScoopInstaller/Scoop#5753] [Help and Questions]
Hi

I just installed Scoop and enabled extras. I am getting this when I search for a package. What is the soltuion?

**Comment by HUMORCE** [comment]
```
scoop bucket rm extras; scoop bucket add extras
```

OR

If you know git, cd to `Scoop\buckets\extras` dir, run `git status` and fix them.
